### PR TITLE
Add module to create oauth tokens

### PR DESCRIPTION
**Pull request by mathias-baumann-sociomantic**

### CODE REVIEW

**Review by mihails-strasuns-sociomantic — APPROVED** [review]
LGTM but would be nice if @leandro-lucarella-sociomantic could have a look as he is more familar with github oauth.

File: src/octod/api/authorizations.d [hunk 1]
```diff
@@ -0,0 +1,58 @@
+/*******************************************************************************
+
+    Provides wrappers on top of some methods documented in
+    https://developer.github.com/v3/oauth_authorizations/
+
+    Copyright: Copyright (c) 2016 Sociomantic Labs GmbH. All rights reserved
```
Comment (leandro-lucarella-sociomantic) [inline src/octod/api/authorizations.d]: Maybe is a good opportunity to start using the correct case for the copyright :D (sociomantic labs GmbH).

**Review by leandro-lucarella-sociomantic — APPROVED** [review]
Just that trivial comment, other than that LGTM.

**Comment by mathias-baumann-sociomantic** [comment]
fixed copy right

**Pull request merged**